client/src/app.config.router.js

```diff
@@ -4,6 +4,7 @@ const homeItemComponent = require('./states/student/home/component');
 const studentLecturesListComponent = require('./states/student/lecturesList/component');
 const studentComponent = require('./states/student/component');
 const projectComponent = require('./states/student/project/component');
+const editComponent = require('./states/student/edit/component');
```

Comment:
please rename it to "editWhat"(folder,component, etc.) as I understand editProfile
because we will have few edits in future and it can be a little bit complicated.. 
everything else is good for me